gatsby-node.ts

```diff
@@ -17,19 +20,52 @@ export const onCreateWebpackConfig = ({ actions }) => {
 export const createSchemaCustomization = ({ actions: { createTypes } }) => {
   createTypes(`
     type MetaJson implements Node {
-      geoJson: GeometryJson @link(from: "jsonId", by: "name")
-      staticMap: File @link(from: "jsonId", by: "name")
+      geoJson: GeometryJson! @link(from: "jsonId", by: "jsonId")
+      staticMap: File! @link(from: "jsonId", by: "name")
     }
   `);
 };
 
+export const onPostBootstrap = ({ getNodesByType }) => {
+  const nGeometries = getNodesByType('GeometryJson').length;
+  const nMeta = getNodesByType('MetaJson').length;
+  if (nGeometries !== nMeta) {
+    throw new Error(
+      `The provided number of geometries (${nGeometries}) does no match the number of meta information (${nMeta})`
+    );
```

Comment:
Maybe we should use the `reporter` from  Node API from Gatsby , see https://www.gatsbyjs.com/docs/reference/config-files/node-api-helpers/#reporter
I'm afraid of loosing this line in the log.